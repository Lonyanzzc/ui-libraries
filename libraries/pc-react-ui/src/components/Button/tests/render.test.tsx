import React from 'react';
import { expect, test } from 'vitest';
import renderer from 'react-test-renderer';
import { Button } from 'antd';

test('Render Default Button', () => {
  const wrapper = renderer.create(<Button />);
  expect(wrapper.toJSON()).toMatchSnapshot();
});
