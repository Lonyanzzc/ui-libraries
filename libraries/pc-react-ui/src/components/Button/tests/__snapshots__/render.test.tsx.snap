// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Render Default Button 1`] = `
<button
  className="ant-btn css-dev-only-do-not-override-2rgkd4 ant-btn-default"
  disabled={false}
  onClick={[Function]}
  style={{}}
  type="button"
/>
`;
